src/sql/first.sql

```diff
@@ -0,0 +1,35 @@
+CREATE DATABASE hospital;
+
+CREATE TABLE diseases
+(
+    id          	INT UNSIGNED  AUTO_INCREMENT NOT NULL PRIMARY KEY,
+    nameDiseases    VARCHAR NOT NULL
```

Comment:
Please use snake_case for the column names

Reply:
Please rename 'nameDiseases' to just 'name' and maybe better to use UNIQUE for this column